packages/govern-subgraph/src/Govern.ts

```diff
@@ -0,0 +1,112 @@
+import { Address, ethereum } from "@graphprotocol/graph-ts";
+import {
+  Executed as ExecutedEvent,
+  Frozen as FrozenEvent,
+  Granted as GrantedEvent,
+  Revoked as RevokedEvent,
+} from "../generated/templates/Govern/Govern";
+import {
+  Action as ActionEntity,
+  Govern as GovernEntity,
+  Execution as ExecutionEntity,
+} from "../generated/schema";
+import { frozenRoles, roleGranted, roleRevoked } from "./lib/MiniACL";
+
+export function handleExecuted(event: ExecutedEvent): void {
+  const govern = loadOrCreateGovern(event.address);
+
+  const execution = loadOrCreateExecution(event);
+
+  createActions(event);
+
+  // add the execution
+  const currentExecutions = govern.executions;
+  currentExecutions.push(execution.id);
+  govern.executions = currentExecutions;
+
+  execution.save();
+  govern.save();
+}
+
+// MiniACL Events
```

Comment:
Using the above pattern the logic for the ACL events was simplified quite a bit. I still want to double check if some of the syntax that I use (like mappings, filters, etc) has not problem during the subgraph deployment.